### ISSUE TITLE
[Feat]: Anomaly rate per section

### ISSUE THREAD

**Issue by shyamvalsan** [netdata/netdata#13513]
### Description

There should be a way to fetch the anomaly rate per family or per section. The user should be able to see the anomaly rate of only metrics related to a particular collector if they so wish.

Eg. 10% of Postgres metrics are anomalous right now.

### Importance

really want

### Value proposition

1. Improved usability
2. Faster monitoring and troubleshooting

This is required for implementing:
- https://github.com/netdata/netdata-cloud/issues/546
- https://github.com/netdata/product/issues/2795

**Comment by shyamvalsan** [comment]
cc: @MrZammler @andrewm4894 @sashwathn

**Comment by ktsaou** [comment]
This is interesting...

**Comment by andrewm4894** [comment]
We had talked about the ability to add "Saved Views" onto the AA tab in some way such that you could just filter the space of stuff being looked at in the AA tab. 

Idea here was if you keep seeing things you dont care about in the AA tab results then you could just build up some sort of saved filter or something. 

So "Postgress" for example could just be a simple pattern or regex or something over charts or contexts. 

Something flexible such that a user could define thier own one and maybe NC also has a list of precanned onces that we maintain too etc. 

So some way from within the filter bar i suppose to just select various "Chart Filters" etc and maybe define and save your own ones. 
![image](https://user-images.githubusercontent.com/2178292/184904823-57936c6d-a810-48c2-b5a0-065bf452f653.png)

Am sure are other ways to do this too but i think some idea or concept of "Chart Filters" or "Saved Views" or something could be fairly easy to do since we can generally filter first and then work out anomaly rates etc on whatever slice of the data a user wants to search over. 

So i think most of the hard work here is in UI/UX of this all i think.

**Comment by andrewm4894** [comment]
maybe its just a way for users to define and save netdata simple patterns that can then bew applied like a filter in some easy way. 

https://learn.netdata.cloud/docs/agent/libnetdata/simple_pattern

**Comment by sashwathn** [comment]
@andrewm4894 : I agree with Shyam above and for every section in our Overview, we can probably create default "views" to display the anomaly rate per section and also expose these views in the AA tab for power users to slide and dice.
